setup.py

```diff
@@ -1,86 +1,96 @@
 """
-Setup Module to setup Python Handlers for the jupyterlab-snippets extension.
+jupyterlab-snippets setup
 """
-import os
+import json
+from pathlib import Path
 
-from setupbase import (
-    create_cmdclass, install_npm, ensure_targets,
-    combine_commands, ensure_python, get_version,
+from jupyter_packaging import (
+    create_cmdclass,
+    install_npm,
+    ensure_targets,
+    combine_commands,
+    skip_if_exists
 )
 import setuptools
 
-HERE = os.path.abspath(os.path.dirname(__file__))
+HERE = Path(__file__).parent.resolve()
 
 # The name of the project
-name="jupyterlab-snippets"
+name = "jupyterlab_snippets"
```

Comment:
Should we keep `jupyterlab-snippets` here? (purely for esthetics reasons)

And maybe use:

```python
package = name.replace('-', '_')
```

For the folder and package names.

Reply:
Thanks for the suggestion. I made the modification